<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositWindow</class>
 <widget class="QWidget" name="DepositWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>888</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartCalc v2.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: ubuntu;
	font-size: 30px;
	font-weight: 0px;
	color: white;
	text-align: center;
	vertical-align: center;
	border-radius: 22px;
	border: 0px;
}

QWidget {
	background-color: #17181a;
}

QPushButton {
	background-color: #1991ff;
	color: #b2daff;
}

QPushButton:pressed {
	background-color: #004787;
	color: #1a7cc0;
}

QSpinBox {
	border-radius: 15px;
	background-color: #303136;
	color: #24a5ff;
	padding-left: 10px;
	padding-right: 10px;
}

QDoubleSpinBox {
	border-radius: 15px;
	background-color: #303136;
	color: #24a5ff;
	padding-left: 10px;
	padding-right: 10px;
}

QDateEdit {
	border-radius: 15px;
	background-color: #303136;
	color: #24a5ff;
	padding-left: 10px;
	padding-right: 10px;
}

QComboBox {
	border-radius: 15px;
	background-color: #303136;
	color: #24a5ff;
	padding-left: 10px;
	padding-right: 10px;
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font-size: 25px;
	border-radius: 28px;
	background-color: #303136;
	color: #24a5ff;
}

QPushButton:pressed {
	background-color: #004787;
	color: #1a7cc0;
}

QPushButton:disabled {
	background-color: #868686;
	color: #5a5a5a;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="calcButton">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>0</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton:indicator {
	background-color: #17181a;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/calc_off.png</normaloff>:/icons/icons/calc_off.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="graphicButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton:indicator {
	background-color: #17181a;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/graphic_off.png</normaloff>:/icons/icons/graphic_off.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="creditButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton:indicator {
	background-color: #17181a;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/credit_off.png</normaloff>:/icons/icons/credit_off.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="depositButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton:indicator {
	background-color: #17181a;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/deposit_on.png</normaloff>:/icons/icons/deposit_on.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>depositButton</zorder>
   <zorder>calcButton</zorder>
   <zorder>creditButton</zorder>
   <zorder>graphicButton</zorder>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="timeframe">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>-999999999</number>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Timeframe (m.)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tax</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="capitalisation">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Capitalization</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculate_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>390</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimumDate">
    <date>
     <year>2024</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>d M yyyy</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="events_group">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>130</y>
     <width>551</width>
     <height>741</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
	font-family: ubuntu;
	font-size: 22px;
	font-weight: 0px;
	color: white;
	text-align: center;
	vertical-align: center;
	border-radius: 16px;
}

QPushButton {
	padding-left: 10px;
	padding-right: 10px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>530</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="events_list">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="supplementions_list">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="add_supplementions">
         <property name="text">
          <string>add supplementions</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="eliminations_list">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="add_eliminations">
         <property name="text">
          <string>add eliminations</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QCheckBox" name="early_closure">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Early closure</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Frequency</string>
   </property>
  </widget>
  <widget class="QComboBox" name="frequency">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>460</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>monthly</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>everyday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>quartaly</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yearly</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="closure_date">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>70</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimumDate">
    <date>
     <year>2024</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>d M yyyy</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="amount">
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>110</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>10000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="tax">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>10000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rate">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>50.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>519</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Total amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>639</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Payments</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>579</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Tax deduction</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="payments_list">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>650</y>
     <width>271</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
	text-align: right;
}

QTableWidget QTableCornerButton::section {
    background-color: #646464;
    border: 1px solid #fffff8;
}

QHeaderView::section {
    background-color: #303136;
    padding: 4px;
    font-size: 14pt;
}</string>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QLabel" name="total_amount">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>519</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="tax_deduction">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>579</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_output">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>501</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
	color: red;
	font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
